slides/index.html

```diff
@@ -1,80 +1,89 @@
 <html>
```

Comment:
any reason you reformatted the entire file? Makes reading the diff very difficult :(